client/my-sites/checkout/src/components/akismet-pro-quantity-dropdown/index.tsx

```diff
@@ -0,0 +1,95 @@
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent, useEffect, useCallback, useState } from 'react';
+import FormLabel from 'calypso/components/forms/form-label';
+import FormSelect, { FormSelectProps } from 'calypso/components/forms/form-select';
+import type { AkismetProQuantityDropDownProps } from './types';
+
+const AkismetSitesSelectHeading = styled.div`
+	font-size: 1.1em;
```

Comment:
I think we should probably use `rem` here right? I guess I am not sure but that's generally what I see being used

Reply:
Yeah I'm not sure... I was thinking that too, but looking at the rest of the checkout package, it seems to be using `em`'s . For example, all throughout this file: https://github.com/Automattic/wp-calypso/blob/trunk/packages/wpcom-checkout/src/checkout-line-items.tsx#L89

Reply:
`rem` is probably more ideal to ensure a consistent styling across the interface, but I think `em` was just used enough that it took root in the package. This could be something we consider going forward with the checkout redesign.